shelter/remoteattestation/ratlsclient.c

```diff
@@ -105,24 +107,22 @@ int ra_tls_echo(int sockfd, unsigned char* mrenclave, unsigned char* mrsigner)
 		goto err_ssl;
 	}
 
-	fprintf(stdout, "wolfSSL_connect success.\n");
 	WOLFSSL_X509 *srvcrt = wolfSSL_get_peer_certificate(ssl);
-	fprintf(stdout, "wolfSSL_get_peer_certificate success.\n");
+
 	int derSz;
 	const unsigned char *der = wolfSSL_X509_get_der(srvcrt, &derSz);
-	fprintf(stdout, "wolfSSL_X509_get_der success.\n");
-
-	sgx_quote_t quote;
-	get_quote_from_cert(der, derSz, &quote);
-	fprintf(stdout, "get_quote_from_cert success.\n");
-	sgx_report_body_t *body = &quote.report_body;
-	printf("  . MRENCLAVE  = ");
+	sgx_report_body_t *body = NULL;
+	uint8_t quote_buff[8192] = {0,};
```

Comment:
Please define 8192 as a contsant rather than hard coding.

Reply:
Ok, change this one but keep the below one.